host-controller/src/main/java/org/jboss/as/host/controller/model/host/HostResourceDefinition.java

```diff
@@ -300,7 +304,16 @@ protected boolean requiresRuntime(OperationContext context) {
 
         });
         hostRegistration.registerReadWriteAttribute(ORGANIZATION_IDENTIFIER, null, new ModelOnlyWriteAttributeHandler(ORGANIZATION_IDENTIFIER));
-        hostRegistration.registerReadOnlyAttribute(DOMAIN_ORGANIZATION_IDENTIFIER, null);
+        // provide the domain-organization, this was defined here, but never had any handlers or storage defined.
+        hostRegistration.registerReadOnlyAttribute(DOMAIN_ORGANIZATION_IDENTIFIER, new ReadAttributeHandler() {
+                    @Override
+                    public void execute(OperationContext context, ModelNode operation) throws OperationFailedException {
+                        ModelNode rootModel = context.readResourceFromRoot(PathAddress.EMPTY_ADDRESS,false).getModel();
+                        final String domainOrganization = (rootModel.hasDefined(DOMAIN_ORGANIZATION)) ? rootModel.get(DOMAIN_ORGANIZATION).asString() : "undefined";
+                        context.getResult().set(domainOrganization);
```

Comment:
This should just be 

context.getResult(rootModel.get(DOMAIN_ORGANIZATION));

We don't use the String "undefined" to mean an undefined node.